css/selectors/descendant.json

```diff
@@ -9,12 +9,28 @@
             "webview_android": {
               "version_added": true
             },
-            "chrome": {
-              "version_added": "1"
-            },
-            "chrome_android": {
-              "version_added": true
-            },
+            "chrome": [
+              {
+                "version_added": true
+              },
+              {
+                "version_added": "61",
+                "notes": [
+                  "<code>&gt;&gt;&gt;</code> is aliased to this selector <a href='https://developers.google.com/web/updates/2017/06/chrome-60-deprecations#make_shadow-piercing_descendant_combinator_behave_like_descendent_combinator'>since its use as the 'shadow-piercing descendant combinator' was deprecated</a>."
+                ]
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": true
```

Comment:
This should go after the `"version_added": "61"` entry, and have `"version_removed": "61"` added to it.